### ISSUE TITLE
create page to help make teams with 20+ skater rosters workable for statsbook output

### ISSUE THREAD

**Issue by erevrav** [dhmay/jamstats#104]
**What version of the tools are you running, and how?**
windows version 1.0.2 with drag-and-drop

**Is your feature request related to a problem? Please describe.**
Our teams often field more than 20 skaters, but 20 is the “official” roster limit for WFTDA/JRDA. They're all in the scoreboard software, and we rarely indicate which skaters will actually be skating. This can yield less-than-helpful statsbook xls files (like the penalty tracking or Game Summary tabs), since the workbook is constructed assuming only the first 20 skaters are real; the high-numbered ones just don't show up in the excel sheet if there's too many. 

**Describe the solution you'd like**
There's a couple of things that would make life easier:

- Callout on some page (it doesn't have to be the first one) when number of rostered skaters for one or both teams in a game is > 20. Also call out (as we do on page 1 today) when number of actually-played is still <20, since the roster can be manually cleaned up so the statsbook usable, but when played >20, it can't be.

- related, have a page with a numbered list of skater numbers & names (sorted alphabetically) by skater number: 007, 10, 100, 1001, 2, etc., ), so I can go back in CRG more-or-less easily and remove skaters who didn't play. Maybe strikethrough the ones who didn't actually play (e.g., got recorded as jammer/pivot/blocker) to make this easier? 

**Describe alternatives you've considered**
Asking the coaches to roster fewer than players? Making 8-year-olds on teams with 23 skaters cry?

**Additional context**

**Comment by dhmay** [comment]
The make-8-year-olds-cry alternative is compelling, but I'll see what I can do.

**Comment by dhmay** [comment]
I'm thinking:
1. Add "skaters rostered" to the team summary table
2. Add a new table for each team with lexicographically sorted skaters, with a column indicating whether they played. I bet there's other information that would be useful there... I could add a penalty count, # jams played, # jams jammed....

**Comment by erevrav** [comment]
yup, those sounds great. (and maybe it will encourage the coaches who see it to give me updated rosters. :) )
